eng/packages/http-client-csharp-mgmt/package.json

```diff
@@ -37,29 +37,27 @@
     "dist/**"
   ],
   "dependencies": {
-    "@azure-typespec/http-client-csharp": "1.0.0-alpha.20250429.2"
+    "@azure-typespec/http-client-csharp": "1.0.0-alpha.20250507.2"
```

Comment:
why is this version not synced with the generator version?

Reply:
I forgot to update this.
Will update